CNX-8892/Sdk/Speckle.Autofac/DependencyInjection/AutofacContainer.cs

```diff
@@ -13,10 +13,8 @@ namespace Speckle.Autofac.DependencyInjection;
 // POC: wrap the IContainer or expose it?
 public class AutofacContainer
 {
-  public delegate void PreBuildEventHandler(object sender, ContainerBuilder containerBuilder);
-
   // Declare the event.
-  public event PreBuildEventHandler PreBuildEvent;
+  public event EventHandler<ContainerBuilder> PreBuildEvent;
```

Comment:
This was another interesting change to reduce boiler plate @BovineOx 
Not sure what you think about this, would be happy to revert if prefered.

Reply:
Right, this looks fine, I confess I think I've used this syntax before but event syntax I haven't used in ages tbh